de.fu_berlin.inf.dpp/test/framework/stf/src/de/fu_berlin/inf/dpp/stf/server/rmi/superbot/component/view/impl/Views.java

```diff
@@ -24,12 +30,34 @@ public static Views getInstance() {
     return INSTANCE;
   }
 
+  private boolean forceUseHtmlGui = false;
+
+  /* Use this method to force the STF to use the HTML-GUI.
```

Comment:
This doesn't look auto-formatted. Also: Such comments belong in the interface.

Reply:
fixed in the second commit